src/waterfall/polarplot.h

```diff
@@ -102,22 +102,9 @@ class PolarPlot : public Waterfall
     void updateMouseColumnData();
 
     QImage _image;
-    float _mouseSampleConfidence;
-    float _mouseSampleDepth;
+    float _mouseSampleAngle;
+    float _mouseSampleDistance;
     QPainter *_painter;
     static uint16_t _angularResolution;
     QTimer* _updateTimer;
-
-    /**
-     * @brief Depth and Confidence package
-     *
-     */
-    struct DCPack {
-        float initialDepth;
-        float length;
-        float confidence;
-        float distance;
-    };
-
-    QVector<DCPack> _DCRing;
```

Comment:
We will need to scale previous data if the range changes. We may be able to accomplish this without a vector of past data by performing a transform on the current image. We can also do a redraw and erase previous data for the first solution because range will not change automatically/frequently.

Reply:
Right now we don't have confidence information, we can do a better approach removing/changing the structure for ping360. Waiting hardware integration to add distance/depth indirect calculation.